proto/ojo/airdrop/v1/airdrop.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+package ojo.airdrop.v1;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/ojo-network/ojo/x/airdrop/types";
+
+option (gogoproto.goproto_getters_all) = false;
+
+// Params defines the parameters for the airdrop module.
+message Params {
```

Comment:
Are we planning on using the params module here? Most of cosmos has been moving away from this

Reply:
No, we should not, good call.